gammapy/scripts/image_pipe.py

```diff
@@ -2,56 +2,196 @@
 
-    def make_background(self):
         log.info('Making background image ...')
 
+    def make_background_normalized_offcounts(self, obs_id, exclusion_mask):
+        """Normalized the background compare to te events in the counts maps outside the exclusion maps
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+            Exclusion regions
+
+
+        """
+        self.make_counts(obs_id)
+        self.make_background(obs_id)
+        counts_sum = np.sum(self.maps["counts"].data * exclusion_mask.data)
+        bkg_sum = np.sum(self.maps["bkg"].data * exclusion_mask.data)
+        scale = counts_sum / bkg_sum
+        self.maps["bkg"].data = scale * self.maps["bkg"].data
+        self.maps['exclusion'] = exclusion_mask
+
+    def make_total_bkg(self, exclusion_mask):
+        """Stack the total bkg from the observation in the 'DataStore'
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+            Exclusion regions
+
+        """
+        self.maps["total_bkg"] = SkyMap.empty_like(self.image)
+        for obs_id in self.data_store.obs_table['OBS_ID']:
+            self.make_background_normalized_offcounts(obs_id, exclusion_mask)
+            self.maps["total_bkg"].data += self.maps["bkg"].data
+
+    def make_significance(self, radius, counts_image=None, bkg_image=None):
+        """Make the dignificance image from the counts and kbg images.
+
+        Parameters
+        ----------
+        radius : float
+            Disk radius in pixels.
+        counts_image : `~gammapy.image.SkyMap`
+            count image
+        bkg_image : `~gammapy.image.SkyMap`
+            bkg image
+
+        """
+
+        if not counts_image:
+            self.maps["significance"] = SkyMap.empty_like(self.image)
+            counts_image = self.maps["total_counts"]
+        else:
+            self.maps["significance"] = SkyMap.empty_like(counts_image)
+        if not bkg_image:
+            bkg_image = self.maps["total_bkg"]
+
+        counts = disk_correlate(counts_image.data, 10)
+        bkg = disk_correlate(bkg_image.data, 10)
+        s = significance(counts, bkg)
+        self.maps["significance"].data = s
+        log.info('Making significance image ...')
+
+    def make_maps(self, exclusion_mask, radius):
+        """Compute the counts, bkg, exlusion_mask and significance images for a set of observation
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+            Exclusion regions
+        radius : float
+            Disk radius in pixels for the significance map.
+
+        """
+        self.make_total_counts()
+        self.make_total_bkg(exclusion_mask)
+        self.make_significance(radius)
+
     def make_psf(self):
         log.info('Making PSF ...')
 
+    def make_exposure(self):
+        log.info('Making Exposure ...')
+
     def fit_source(self):
```

Comment:
@JouvinLea - I'd say remove the `fit_source` method here.

Modeling / fitting is already partly implemented elsewhere, and even if we do add a convenience "maker" class for it, it would be more flexible to have a separate one, no?
So the job of this `ImageAnalysis` class is to prepare the images that go in the likelihood fit (if that's what the user wants, it could also just go e.g. to TS map computation and source finding).